setup.py

```diff
@@ -43,6 +43,11 @@
     #units
     'timm',
     'smart_open',
+
+    #timesfm
+    "timesfm[torch]>=1.2.0,<1.5;python_version>='3.11'",
+    'jax',
```

Comment:
please add the `python 3.11` requirement for all dependency packages for timesfm.

```
"jax;python_version>='3.11'"
```

also please include a range of `jax` versions that are compatible.